assets/sass/patterns/molecules/button-collection.scss

```diff
@@ -1,21 +1,23 @@
 @import "../../definitions";
 
-$gutter: 10px;
+$gutter: 10;
 
 .button-collection {
   display: table;
+  @include nospace();
+  @include blg-spacing("bottom", "small", "margin");
+  @include margin(-$blg-space-small-in-px, "top");
```

Comment:
With all the hyphens in variable names, `@include margin($blg-space-small-in-px * -1, "top");` is easier to read here.